src/main/javascript/prepareRelease.js

```diff
@@ -31,4 +32,10 @@ if (!swagger.info.version.includes('-SNAPSHOT')) {
     });
 
     fsx.copySync(__dirname + '/src/models/models-readme.md', resourcesFolder + '/javascript/readme.md');
+
+    // Generate graphQL documentation
+    let packageJson = __dirname + '/../resources/javascript/package.json';
+    let graphqlSchema = __dirname + '/../resources/graphql/schema.graphql';
+    let docsFolder = __dirname + '/../../../docs/graphql';
+    cp.execSync(`$(npm bin)/graphdoc --force -c ${packageJson} -s ${graphqlSchema} -o ${docsFolder}`);
```

Comment:
You might want to add `, {stdio: 'inherit'}` here so we can see the output of the script during the release.